### ISSUE TITLE
Unable to find note type from MPVacious

### ISSUE THREAD

**Issue by ryanbrownnetworking777** [Ajatt-Tools/mpvacious#55]
1) OS name  Windows 10 Pro
2) OS version 20H2
3) mpv version 0.33.0 (10/17/2021.7z unofficial zip file)
4) Logs
5) Screenshots, if applicable

**Describe the bug**
A description of what the bug is.

Note type is not found when pressing 'n' and exporting from MPVacious

**To Reproduce**

1) Load up a video in MPV
2) Add subtitle file
3) Add timings with c
4) turn on clipboard autocopy with t
5) export note with n
6) see error 'unable to find notetype'

**Expected behavior**
A description of what you expected to happen.

Expected to export note correctly when pressing n. 

**Additional commentary**

I noticed in a prior run through tutorial video [here](https://www.youtube.com/watch?v=tkFxnY0mehE) around the 15:56 mark that the anki_user config was set in his subs2srs.lua file. However around mid december of last year that config was removed from the script and all references to the collection itself was removed. I'm clearly missing something here. Any guidance is appreciated!

**subs2srs.conf**
deck_name="日本語::文章"
model_name="文章"
snapshot_quality=100
audio_bitrate="32k"
note_tag="subs2srs %n %t"
use_forvo="yes"

(This issue happened when I had the default name of "Japanese sentences" used as well)

**Comment by tatsumoto-ren** [comment]
Quotes aren't needed when editing subs2srs.conf. See [this example](https://github.com/Ajatt-Tools/mpvacious/blob/master/.github/RELEASE/subs2srs.conf). 

This should work:
```
model_name=文章
```

**Comment by ryanbrownnetworking777** [comment]
Thank you so much your quick reply. I updated my .conf file in accordance with your suggestions and I have moved forward to a new error! Now it's saying 'cannot create note because it is empty'. I'm still missing something here lol but it's always good to get new errors.

**Comment by tatsumoto-ren** [comment]
Set `sentence_field` to the name of the sort field of your note type.

**Comment by ryanbrownnetworking777** [comment]
![image](https://user-images.githubusercontent.com/39069157/139747328-4c92731a-d209-43c4-a86c-7df8274ff90b.png)
![image](https://user-images.githubusercontent.com/39069157/139747368-bbba44e8-6a13-4af9-b2bc-585787846ebd.png)
I believe I have and I'm still seeing this issue. Sadly, I am still missing something here.

**Comment by ryanbrownnetworking777** [comment]
I'm using the defaults.

**Comment by tatsumoto-ren** [comment]
I think the sort field should also be the first field of a note type. Otherwise you get the following error when adding notes. 

![screenshot-2021-11-01-22-39-48](https://user-images.githubusercontent.com/69171671/139751371-2052c96c-7f57-412d-b895-f64934688833.png)

**Comment by ryanbrownnetworking777** [comment]
Okay, I will review that.

**Comment by ryanbrownnetworking777** [comment]
Okay I figured out what was going on. The tutorial I followed had a workflow that had yomichan enter the card and then use mpvacious to update it. That's why pressing n doesn't work; but pressing m does. Thank you very much for the context here; if I want to use n to create the note  I need to have the sort field be the first field and have it set to the sentence export field. Otherwise I'll have to use m. I get it now and have made an example card using this workflow with mpvacious. Thank you for your guidance here in helping me set this up.
